InnerEye/ML/model_training.py

```diff
@@ -141,12 +139,15 @@ def create_lightning_trainer(container: LightningContainer,
         logging.info("Adding monitoring for GPU utilization")
         callbacks.append(GPUStatsMonitor(intra_step_time=True, inter_step_time=True))
     # Add the additional callbacks that were specified in get_trainer_arguments for LightningContainers
-    if "callbacks" in kwargs:
-        more_callbacks = kwargs.pop("callbacks")
+    additional_args = container.get_trainer_arguments()
+    # Callbacks can be specified via the "callbacks" argument (the legacy behaviour) or the new get_callbacks method
+    if "callbacks" in additional_args:
```

Comment:
Is the idea to eventually phase out the legacy get_trainer_arguments method? Is there any reason for still keeping it at the moment?

Reply:
backwards compatibility, yes